### PR TITLE
Update of scripts during functional testing

### PR DESCRIPTION
**Pull request by dboltovskyi**

### CODE REVIEW

File: test_scripts/API/AddSubMenu/ATF_AddSubMenu_Negative_SubMenuIcon_OutUpperBound.lua [hunk 1]
```diff
@@ -54,7 +54,7 @@ function Test:Precondition_ActivateApp()
   end)
   EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN", audioStreamingState = "AUDIBLE"})
 end
-commonSteps:PutFile("PutFile_strOutofBoundFileName", strOutofBoundFileName)
+-- commonSteps:PutFile("PutFile_strOutofBoundFileName", strOutofBoundFileName)
```
Comment (istoimenova) [inline test_scripts/API/AddSubMenu/ATF_AddSubMenu_Negative_SubMenuIcon_OutUpperBound.lua]: @dboltovskyi, if the script will not use PutFile() as precondition, please delete this line.

Comment (istoimenova) [inline test_scripts/API/AddSubMenu/ATF_AddSubMenu_Negative_SubMenuIcon_OutUpperBound.lua]: @dboltovskyi, The test AddSubMenu_SubMenuIconOutUpperBound should verify that within default timeout(10 sec) OnHashChange will not come.
When using Times(0), ATF doesn't wait default timeout to expire.
Please add "commonTestCases:DelayedExp(10000)"

Comment (istoimenova) [inline test_scripts/API/AddSubMenu/ATF_AddSubMenu_Negative_SubMenuIcon_OutUpperBound.lua]: @dboltovskyi, the script is checking that case when SDL should not process request to HMI, so well be better to add aditional in test AddSubMenu_SubMenuIconOutUpperBound: 
EXPECT_HMICALL("UI.AddSubMenu"):Times(0)

**Comment by dboltovskyi** [comment]
@istoimenova , @maria-mih, This and a few more scripts updated according to comments provided. Please check  	5af213b

**Comment by dboltovskyi** [comment]
@istoimenova, Scripts were updated, please check 6dc7bdd

**Comment by dboltovskyi** [comment]
@istoimenova, @maria-mih Please review  8f23c3d
Changes:
1) Scripts renamed using counter as suffix (test set also updated)
2) Removed useless usage of absolute paths in requests
3) Removed usage of Times(1) function

File: test_scripts/API/AddSubMenu/015_ATF_AddSubMenu_Positive_IconMissingInStorage.lua [hunk 2]
```diff
@@ -96,7 +85,7 @@ function Test:AddSubMenu_NoIconInAppStorage()
     subMenuIcon =
     {
       imageType = "DYNAMIC",
-      value = storagePath .. "menuIcon.jpg"
+      value = "menuIcon.jpg"
```
Comment (istoimenova) [inline test_scripts/API/AddSubMenu/015_ATF_AddSubMenu_Positive_IconMissingInStorage.lua]: @dboltovskyi, for "DYNAMIC" imageType, expected value should have full SDL path (for CI storagePath .. "menuIcon.jpg" , locally SDL returns "./storage...", please return verification: "value = storagePath .. "menuIcon.jpg""

For info: when imageType = "STATIC" SDL uses binary and the check is value = "menuIcon.png"

Please note that for now there is SDL defect regarding imageType = "DYNAMIC" that is observed for PerformAudioPassThru: "[GENIVI] SDL sends incorrect path for Dynamic image with audioPassThruIcon"

File: test_scripts/API/AddSubMenu/020_ATF_AddSubMenu_Positive_SubMenuIcon_DYNAMICType.lua [hunk 3]
```diff
@@ -81,7 +78,7 @@ function Test:AddSubMenu_SubMenuIconDYNAMIC()
     subMenuIcon =
     {
       imageType = "DYNAMIC",
-      value = storagePath .. "menuIcon.jpg"
+      value = "menuIcon.jpg"
```
Comment (istoimenova) [inline test_scripts/API/AddSubMenu/020_ATF_AddSubMenu_Positive_SubMenuIcon_DYNAMICType.lua]: @dboltovskyi, for "DYNAMIC" imageType, expected value should have full SDL path (for CI storagePath .. "menuIcon.jpg" , locally SDL returns "./storage...", please return verification: "value = storagePath .. "menuIcon.jpg""

For info: when imageType = "STATIC" SDL uses binary and the check is value = "menuIcon.png"

Please note that for now there is SDL defect regarding imageType = "DYNAMIC" that is observed for PerformAudioPassThru: "[GENIVI] SDL sends incorrect path for Dynamic image with audioPassThruIcon"

**Comment by dboltovskyi** [comment]
@istoimenova , Please check new commit 109511a
Changes:
1) Implemented correct usage of SendError and SendResponse functions depends on success status
2) Added absolute path expectation in SDL to HMI requests in case of DYNAMIC image type
3) Small fixes

**Pull request merged**